lib/getToken.js

```diff
@@ -0,0 +1,18 @@
+const parseToken = (tokString) => {
+	let tok
+	try {
+		tok = tokString.match('\"(.+)\"')[1]
+	} catch (err) {
+		console.log('error parsing out token from Authorization header: ' + err)
+	}
+	return tok
+}
+
+export default function getToken(req) {
```

Comment:
We're now using this in both the `/login` route and in the authentication middleware, so I extracted it.